specification/columns/columns.mdpp

```diff
@@ -23,6 +23,8 @@ The FOCUS specification defines a group of columns that provide qualitative valu
 !INCLUDE "invoiceissuer.md",1
 !INCLUDE "listcost.md",1
 !INCLUDE "listunitprice.md",1
+!INCLUDE "contractedcost.md",1
```

Comment:
nit: We need these to be in alphabetical order i believe.

Reply:
good catch! resolved.